lib/Differentiator/ReverseModeVisitor.cpp

```diff
@@ -897,18 +871,12 @@ Expr* getArraySizeExpr(const ArrayType* AT, ASTContext& context,
     StmtDiff elseDiff = VisitBranch(If->getElse());
 
     Stmt* Forward = clad_compat::IfStmt_Create(
-        m_Context, noLoc, If->isConstexpr(), nullptr, nullptr,
-        condDiff.getExpr(), noLoc, noLoc, thenDiff.getStmt(), noLoc,
-        elseDiff.getStmt());
+        m_Context, noLoc, If->isConstexpr(), nullptr, nullptr, condDiffStored,
```

Comment:
```suggestion
        m_Context, noLoc, If->isConstexpr(), /*Init=*/nullptr, /*Var=*/nullptr, condDiffStored,
```
@vaithak, not sure why clang-tidy refuses to complain but that's quite annoying. Can you take another look?

Reply:
Fixed it.